Tryouts/Plugins/Services/CommunicationsServices/GrpcCommunicationsService/SubscriptionClientImpl.cs

```diff
@@ -43,20 +42,45 @@ public async Task Publish<TMessage>(Topic topic, TMessage msg)
             await _client.PublishAsync(publishRequest);
         }
 
-        public async IAsyncEnumerable<TMessage> Subscribe<TMessage>(Topic topic)
-            where TMessage : IMessage, new()
+        private AsyncServerStreamingCall<ReturnedSubscriptionItem>? _replies;
```

Comment:
Won't you need an instance for each topic? Can a client subscribe to one topic multiple times?

Reply:
yes, client can subscribe for each topic multiple times